### ISSUE TITLE
Improving gender classification

### ISSUE THREAD

**Issue by t3ch9** [serengil/deepface#144]
Hello, I recently ran some tests using the following image which incorrectly classified the gender as a man: 

https://www.sheknows.com/wp-content/uploads/2018/08/strong-black-woman-feature_h42gdn.jpeg

via

img = r'images\strong-black-woman.jpeg'
obj = DeepFace.analyze(img_path = img, actions = ['age', 'gender', 'race', 'emotion'])
print(obj["age"]," years old ",obj["dominant_race"]," ",obj["dominant_emotion"]," ", obj["gender"])

Is there any documentation or suggestions for improving the gender model with more training data?

**Comment by serengil** [comment]
Hello, failing individual test instances is not important in machine learning studies. Gender model got 97.44% accuracy, 96.29% precision and 95.05% which is acceptable. Here you can find its tutorial: https://sefiks.com/2019/02/13/apparent-age-and-gender-prediction-in-keras/ It covers training stages.
